src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -240,14 +242,20 @@ namespace OpenLoco::Ui::Windows::PromptBrowse
         return false;
     }
 
+    // 0x00447174
+    static void freeFileHeader()
+    {
+        call(0x00447174);
```

Comment:
Free `S5::SaveDetails` is that the file header? I would perhaps call it the savePreviewDetails.